dist/tsc/paymasterclient.d.ts

```diff
@@ -0,0 +1,58 @@
+import { ethers, FetchRequest, JsonRpcApiProviderOptions, Networkish, TransactionRequest } from 'ethers';
+import type { AddressLike } from 'ethers/src.ts/address';
+import type { BigNumberish } from 'ethers/src.ts/utils';
+export type IsSponsorableResponse = {
+    Sponsorable: boolean;
+    SponsorName: string;
+    SponsorIcon: string;
+    SponsorWebsite: string;
+};
+export declare enum GaslessTransactionStatus {
+    New = 0,
+    Pending = 1,
+    Confirmed = 2,
+    Failed = 3,
+    Invalid = 4
+}
+export type GaslessTransaction = {
+    readonly TxHash: string;
+    readonly BundleID: bigint;
+    readonly FromAddress?: AddressLike;
+    readonly ToAddress?: AddressLike;
+    readonly RawData: string;
+    readonly Status: GaslessTransactionStatus;
+    readonly GasUsed: bigint;
+    readonly PolicyID: bigint;
```

Comment:
it should be `PolicyUUID`, check the latest code.

Reply:
done